schemas/styles/csl.rnc

```diff
@@ -893,6 +897,48 @@ div {
         "partial-first"
     }?
   
+  ## Options affecting rendering of title variables, for cs:style, cs:citation and cs:bibliography
+  title-inheritable-options =
+    
+    ## Inheritable title option, specify the delimiter between "title-main" and "title-sub".
+    attribute title-delimiter { text }?,
+    
+    ## Inheritable title option, specify the delimiter between multiple subtitles.
+    ## The same as "title-delimiter" if not specified.
+    attribute title-sub-delimiter { text }?,
+    
+    ## Inheritable title option, specify how titles are split into "title-main" and "title-sub".
+    [ a:defaultValue = "simple" ]
+    attribute title-split {
+      
+      ## Matches "."", ":", "::", "!", "?" 
+      "simple"
+      | 
+        ## Matches the values for "simple" plus ";"
+        "extended"
+      | 
+        ## Matches the values for "simple" plus "—" and ";"
+        "full"
+      | 
+        ## Matches the values for "simple" plus ";"" 
+        ## and "[;,] or[,:]" (see CMoS 14.91), e.g.  "Moby-Dick; or, The Whale"
+        "chicago"
+    }?,
+    
+    ## Inheritable title option, specify which delimiters are normalized.
+    [ a:defaultValue = "simple" ]
+    attribute normalize-title-delimiters {
+      
+      ## Normalize "."", ":", "::", "!", "?" 
```

Comment:
since there's duplication here, does this not suggest a need to break out a separate pattern?

Reply:
How would that look like?

Reply:
```
myatt =	attribute my-attribute { myatt.values }
myatt.values = "one" | "two" | "three"
```

So you'd just move the values into a pattern, and then refer to that.

Reply:
Argh, these options are actually identical: simple for splitting is not the same as simple for normalizing. Copy-paste error.
While you split titles at those characters, you only normalize periods and colons (not ? and !)

Reply:
I don't know if this is a bad idea (and sorry if you already considered or mentioned this), but I figure I may as well ask:

What if these values become localized terms, but where you could override locally like so?

``` xml
<style title-sub-delimiter=":?.;">
```

If that could work, this would just be a string.

Or a space-separated list:

``` xml
<style title-sub-delimiter=": ? . ; ::">
```

There's probably disadvantages, but it's very flexible.

Reply:
Not sure about localized, but yeah I've proposed something similar somewhere along the way. Question us probably if reasonable predefined sets will make style coding easier.

One problem again: how to deal with chicago rules for splitting?
@fbennett thoughts?

Reply:
Perhaps this could work
```xml
<style title-sub-delimiter=": ? . ; :: chicago">
```
Or:
```xml
<style title-sub-delimiter=": ? . ; ::" chicago-subtitle-delimiter="true">
```

Reply:
@bwiernik 
You were more in favor of pre-defined sets if I remember correctly. Perhaps can answer that question? I think that a space-separated list may be more flexible, but that we also cover the main combinations with the predefined options, which may be more user-friendly.

Reply:
And to be clear, I'm not sure what's better myself: just want to consider it.